Makefile

```diff
@@ -50,7 +50,8 @@ bootstrap: \
 	build \
 	compile \
 	fixtures \
-	hooks
+	hooks \
+	init-superset
```

Comment:
Is it necessary to include superset in the bootstrap? As it is experimental for now ?